pkg/okta/authentication_middleware.go

```diff
@@ -261,12 +259,10 @@ func (f MiddlewareFactory) NewOktaWebSocketAuthenticationMiddleware(logger *zap.
 		// Parse auth header into JWT object
 		jwt, err := f.jwt(logger, token)
 		if err != nil {
-			fmt.Println("ERROR PARSING JWT", err)
-			// TODO How to error handle here?
+			logger.Warn("could not parse jwt from token", zap.Error(err))
```

Comment:
Outside the scope of this immediate work, we need to evaluate how to gracefully handle this error case.